week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -6,12 +6,12 @@
   They have also decided that they should add a 1% fee to all foreign transactions
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
-
+var fee = 0.01;
```

Comment:
This is better. Even better would be to use the const keyword here for values you don't think will change.